code/game/objects/structures/electricchair.dm

```diff
@@ -73,6 +73,12 @@
 		M.Stun(60)
 		M.Jitter(60)
 		visible_message("<span class='danger'>The electric chair went off!</span>", "<span class='danger'>You hear a deep sharp shock!</span>")
+		var/mob/living/carbon/human/H = null
+		if(ishuman(M))
+			H = M
+			if(isgolem(H)) //golems can't be burned
```

Comment:
I actually just had a look and `burn_skin()` calls `/organ/external/proc/take_damage()` anyway, so I'm getting two birds with one stone.